src/test-utils/cloud-runner-test-helpers.ts

```
@@ -0,0 +1,10 @@
+import { BuildParameters } from '../model';
+import { Cli } from '../model/cli/cli';
+import { OptionValues } from 'commander';
+
+export const TIMEOUT_INFINITE = 1e9;
+
+export async function createParameters(overrides?: OptionValues) {
+  if (overrides) Cli.options = overrides;
+  return BuildParameters.create();
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)